File: sub-web-model/src/main/java/org/github/base/entity/TagEntity.java
package org.github.base.entity;

import org.github.base.Model;
import com.baomidou.mybatisplus.annotation.*;
import java.io.Serializable;
import javax.annotation.Nullable;
import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;
import lombok.Data;
import lombok.EqualsAndHashCode;

/**
 * <p>
 * 
 * </p>
 *
 * @author JYD_XL
 * @since 2021-05-07
 */
@SuppressWarnings("all")
@Data
@EqualsAndHashCode(callSuper = true)
@TableName("tag")
@ApiModel(value="TagEntity对象", description="")
public class TagEntity extends Model<TagEntity> {

  /** UID */
  private static final long serialVersionUID = 1L;

  @TableId(value = "id", type = IdType.AUTO)
  private @Nullable Integer id;

  @TableField("name")
  private @Nullable String name;

  /** 0：未删除	            1：已删除 */
  @ApiModelProperty(value = "0：未删除	            1：已删除")
  @TableField("is_deleted")
  private @Nullable Integer isDeleted;

  @TableField("create_time")
  private @Nullable Long createTime;

  @TableField("last_update_time")
  private @Nullable Long lastUpdateTime;

  @TableField("click_count")
  private @Nullable Long clickCount;

  /** 0：系统标签	            1：用户标签 */
  @ApiModelProperty(value = "0：系统标签	            1：用户标签")
  @TableField("type")
  private @Nullable Integer type;

  @Override
  protected Serializable pkVal() {
    return this.id;
  }

}

File: sub-mysql-blog-entity/src/main/java/org/github/mysql/blog/base/entity/TagEntity.kt
package org.github.mysql.blog.base.entity

import com.baomidou.mybatisplus.annotation.IdType
import com.baomidou.mybatisplus.annotation.TableField
import com.baomidou.mybatisplus.annotation.TableId
import com.baomidou.mybatisplus.annotation.TableName
import java.time.LocalDateTime
import org.github.base.Entity
import io.swagger.annotations.ApiModel
import io.swagger.annotations.ApiModelProperty

/**
 * 
 *
 * @author JYD_XL
 */
@TableName("tb_tag")
@ApiModel(value = "TagEntity对象", description = "")
open class TagEntity : Entity() {

    @TableId(value = "id", type = IdType.AUTO)
    open var id: Int? = null

    /** 标签名 */
    @ApiModelProperty(value = "标签名")
    @TableField("tag_name")
    open var tagName: String? = null

    /** 创建时间 */
    @ApiModelProperty(value = "创建时间")
    @TableField("create_time")
    open var createTime: LocalDateTime? = null

    companion object {

        private const val serialVersionUID = 1L

        const val ID : String = "id"

        const val TAG_NAME : String = "tag_name"

        const val CREATE_TIME : String = "create_time"

    }

    override fun pkVal(): java.io.Serializable? {
        return id
    }

    override fun toString(): String {
        return "TagEntity{" +
        "id=" + id +
        ", tagName=" + tagName +
        ", createTime=" + createTime +
        "}"
    }

}
